src/nbl/video/CVulkanLogicalDevice.h

```diff
@@ -467,6 +465,172 @@ class CVulkanLogicalDevice final : public ILogicalDevice
         return !anyFailed;
     }
 
+    void updateDescriptorSets_impl(uint32_t descriptorWriteCount, const IGPUDescriptorSet::SWriteDescriptorSet* pDescriptorWrites, uint32_t descriptorCopyCount, const IGPUDescriptorSet::SCopyDescriptorSet* pDescriptorCopies)
+    {
+        constexpr uint32_t MAX_DESCRIPTOR_ARRAY_COUNT = 256u;
```

Comment:
since you're using `vector` now, there's no need for this

Reply:
well you should be using `push_back`

Reply:
TODO

Reply:
Resolved.